src/Command/Docker/Build.php

```diff
@@ -233,4 +252,31 @@ public function isEnabled(): bool
     {
         return !$this->environment->isMasterBranch();
     }
+
+    /**
+     * @param array $unsupportedErrorMsg
+     * @param array $emptyVersions
+     * @return string
+     */
+    private function createQuestionMessage(array $unsupportedErrorMsg, array $emptyVersions): string
+    {
+        $questionMessage = '';
+        if ($emptyVersions) {
+            $questionMessage .= sprintf(
+                "Some service versions were not provided:\n" .
+                "Using the versions specified in your configuration files for the following services: %s\n" .
```

Comment:
There are just my thoughts :) : 
 - this message says nothing whether service be installed or not. So, user may just guess or check later.
 - we already know which services are configured in the files ($versionList, line 214) and we may certainly say that something won't be installed.
It's up to you, what you will do (or won't do) after reading this comment :-D